d011-blackjack/Untitled-1.py

```diff
@@ -0,0 +1,128 @@
+#initially, randomly distribute two cards each to house and player 
+#if over 21, lose | if tie, house win | greater point wins otherwise
+#house has less than 16 points, add one card
+#player gets two cards, reveal
+#ask player if want another card
+#
+
+import random 
+card = ['A',
+         2,
+         3,
+         4,
+         5,
+         6,
+         7,
+         8,
+         9,
+        10,
+       'J',
+       'Q',
+       'K']
+    
+house_hand = []
+player_hand = []
+#random.shuffle(card)
+
+house_hand.append(random.choice(card))
+player_hand.append(random.choice(card))
+house_hand.append(random.choice(card))
+player_hand.append(random.choice(card))
+#print(house_hand)
+#print(player_hand)
+#print(player_hand[0])
+
+house_value = 0
+player_value = 0
+
+def Card_Value(card):
```

Comment:
There are many ways to name variables and functions. In python, the convention is _snake_case_, i.e. all words in lowercase, with underscores (`_`) as separator.